app/controllers/questions/start_controller.rb

```diff
@@ -22,5 +24,11 @@ def submit
     def start_form_params
       params.require(:start_form).permit!
     end
+
+    def store_form_in_session!
+      SelectedAnswerStore
+        .new(session)
+        .store_selected_answers(:priority_repair, @form.priority_repair)
```

Comment:
Would it be more explicit/safer to pass in the form as a parameter here? 

If we were in a simple class and `@form` was set in an instance variable then it would always definitely have a value, but in a controller it won't always have a value.

Alternatively, how would you feel about increasing the line length in rubocop and inlining it?

Reply:
Storing the session got bumped out of this card, but I'd already implemented it here, even though it doesn't make sense in this context.

How about I remove saving into the session here altogether, and revisit it when there's a need to do so?